values.yaml

```diff
@@ -53,6 +53,13 @@ kpi:
     minAvailable: 1
   env:
     normal:
+      # By default, the following are set but can be overridden
+      DEFAULT_FROM_EMAIL: "support@kobotoolbox.org"
+      ENKETO_EXPRESS_PUBLIC_SUBDOMAIN: "ee"
+      KOBOCAT_PUBLIC_SUBDOMAIN: "kc"
+      KOBOFORM_PUBLIC_SUBDOMAIN: "kf"
+      PUBLIC_REQUEST_SCHEME: "https"
+      SECURE_HSTS_SECONDS: "3000" # (recommend set to greater value)
```

Comment:
I agree with your recommendation and also with (what I assume is) your reluctance to set a longer default that might cause real trouble for people as they initially configure their instances.

My concern is that having a duration this short (less than 1 hour) doesn't provide much of a security benefit, but having HSTS enabled at all by default might lead to people thinking, "Oh, my instance has HSTS already! No need to do anything there!"

Reply:
Ok, I'll remove it.